.nvmrc

```diff
@@ -1 +1 @@
-v16.14.0
+v20.9.0
```

Comment:
I recommend using `lts/iron` instead so that we automatically use the latest Iron version and not fall behind on minor versions. Should not cause compatibility issues.